lession_03/home_work_03.py

```diff
@@ -0,0 +1,166 @@
+# import re
+#
+# alice_in_wonderland = (
+#     '"Would you tell me, please, which way I ought to go from here?"\n'
+#     '"That depends a good deal on where you want to get to," said the Cat.\n'
+#     '"I don\'t much care where ——" said Alice.\n'
+#     '"Then it doesn\'t matter which way you go," said the Cat.\n'
+#     '"—— so long as I get somewhere," Alice added as an explanation.\n'
+#     '"Oh, you\'re sure to do that," said the Cat, "if you only walk long enough."'
+#
+# )
+# # Task 02: Знайти всі одинарні лапки у тексті
+# single_quotes = re.findall(r"'", alice_in_wonderland)
+#
+# # Task 02: Відобразити всі знайдені одинарні лапки
+# print("Знайдені одинарні лапки:", single_quotes)
+#
+# # Task 03: Вивести змінну на друк
+# print(alice_in_wonderland)
+
+
+"""
+    # Задачі 04 -10:
+    # Переведіть задачі з книги "Математика, 5 клас"
+    # на мову пітон і виведіть відповідь, так, щоб було
+    # зрозуміло дитині, що навчається в п'ятому класі
+"""
+# task 04
+"""
+Площа Чорного моря становить 436 402 км2, а площа Азовського
+моря становить 37 800 км2. Яку площу займають Чорне та Азов-
+ське моря разом?
+"""
+# black_sea_s = 436402
+# azov_sea_s = 37800
+# total_area = black_sea_s + azov_sea_s
+# print(f"Загальна площа Чорного та Азовського морів {total_area}")
+
+
+# task 05
+"""
+Мережа супермаркетів має 3 склади, де всього розміщено
+375 291 товар. На першому та другому складах перебуває
+250 449 товарів. На другому та третьому – 222 950 товарів.
+Знайдіть кількість товарів, що розміщені на кожному складі.
+"""
+# total_sum = 375291
+# first_second_storage = 250449
+# second_third_storage = 222950
+#
+# third_storage = total_sum - first_second_storage
+# second_storage = first_second_storage - third_storage
```

Comment:
hi. pay attention to your calculations to second storage. are you sure that it is correct?